src/Components/src/Microsoft.AspNetCore.Components/ComponentBase.cs

```diff
@@ -262,13 +337,13 @@ void IHandleAfterRender.OnAfterRender()
             {
                 onAfterRenderTask.ContinueWith(task =>
```

Comment:
Ooh, just noticed this. It's my mistake that this `ContinueWith` is still here.

@javiercn Would you be willing to replace this `ContinueWith` with an `await`? The code-as is won't capture the async context and hence might run the callback simultaneously with other lifecycle methods. It's not an immediate disaster since `OnAfterRender` is unlikely to trigger another render, but people may be relying on the idea that all lifecycle method continuations are serialized.

Reply:
No problem

Reply:
Done